bundles/action/org.openhab.action.mail/README.md

```diff
@@ -19,7 +21,7 @@ This action service can be configured via the `services/mail.cfg` file.
 | port          | 25 (resp. 587 for TLS/SSL) | No                                                                      | SMTP port to use                                                                                                                                    |
 | username      |                            | if the SMTP server at `<hostname>` and `<port>` requires authentication | SMTP user name                                                                                                                                      |
 | password      |                            | if the SMTP server at `<hostname>` and `<port>` requires authentication | SMTP password                                                                                                                                       |
-| from          |                            | Yes                                                                     | Email address to use for sending mails                                                                                                              |
+| from          |                            | Yes                                                                     | Email address to use for sending mails. The email address can be given as from=user@domain.com" or from=First Lastname <user@domain.com> (do not use quotes).                                                                                                               |

```

Comment:
See comment above about proper use of left-angle bracket.
Also, please remove the stray double-quote character.

Reply:
Edited - please check and resolve if okay.